### ISSUE TITLE
Commands git.live.showAccountList and git.live.showSettings Not Found

### ISSUE THREAD

**Issue by DanHugoDanHugo** [GitLiveApp/GitLive#129]
Looks like the GitLive extension is not loading completely with this configuration, which is recently updated (ie still works on my laptop, config info in comment below).

## VSCode
Version: 1.70.2
Commit: e4503b30fc78200f846c62cf8091b76ff5547662
Date: 2022-08-16T05:36:25.715Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Linux x64 5.18.0-4-amd64

## GitLive Extension
v15.0.11

I have tried uninstall/reinstall/disable/enable etc but no deeper dive than that and the version comparison.

**Comment by DanHugoDanHugo** [comment]
Working config (completely unchanged otherwise, still working as expected compared to config reported):

## VSCode
Version: 1.70.1
Commit: 6d9b74a70ca9c7733b29f0456fd8195364076dda
Date: 2022-08-10T06:09:15.055Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Linux x64 5.18.0-4-amd64

## GtLive Extension
v15.0.11

**Comment by Ygnys** [comment]
Hey @DanHugoDanHugo, thanks for reporting! We are working on reproducing the issue, will update you as soon as we find a fix for this.

**Comment by Ygnys** [comment]
Hey @DanHugoDanHugo unfortunately so far we were not able to recreate the issue. Have you maybe updated your version of Linux recently? Also, does this happen with any repository you're trying to open? Thanks!

**Comment by DanHugoDanHugo** [comment]
You can see in the issue submission, versus my first comment, the configuration delta.

Occurs 100%.

**Comment by Ygnys** [comment]
Could you check the console for errors (Help > Toggle Developer Tools) and share the output? If you prefer you can send it by email to support@git.live

**Comment by DanHugoDanHugo** [comment]
Can you include your testing configuration so we can compare?

Here is the relevant console output:

Activating extension 'TeamHub.teamhub' failed: $.update is not a function.
$onExtensionActivationError @ mainThreadExtensionService.ts:111
_doInvokeHandler @ rpcProtocol.ts:455
_invokeHandler @ rpcProtocol.ts:440
_receiveRequest @ rpcProtocol.ts:370
_receiveOneMessage @ rpcProtocol.ts:296
(anonymous) @ rpcProtocol.ts:161
invoke @ event.ts:622
deliver @ event.ts:824
fire @ event.ts:785
fire @ ipc.net.ts:638
_receiveMessage @ ipc.net.ts:959
(anonymous) @ ipc.net.ts:832
invoke @ event.ts:622
deliver @ event.ts:824
fire @ event.ts:785
acceptChunk @ ipc.net.ts:382
(anonymous) @ ipc.net.ts:338
E @ ipc.net.ts:70
emit @ node:events:390
addChunk @ node:internal/streams/readable:315
readableAddChunk @ node:internal/streams/readable:289
Readable.push @ node:internal/streams/readable:228
onStreamRead @ node:internal/stream_base_commons:199
console.ts:137

**Comment by Steakeye** [comment]
@DanHugoDanHugo, the following VSCode config works on my setup and GitLive `15.0.11`
```
Version: 1.70.2
Commit: e4503b30fc78200f846c62cf8091b76ff5547662
Date: 2022-08-16T05:36:25.715Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Linux x64 5.15.0-25-generic snap
```

**Comment by DanHugoDanHugo** [comment]
That's close enough for horseshoes.

Was working as expected (viewing team members, UI elements functioning) with 1.70.1 on my laptop, when I updated code to 1.70.2 it fails just as it does on my main dev machine (original issue submission). 

When I disable all extensions and enable only GitLive (plus the extensions that run regardless), so I applied my old school QA powers and luckily it was easy to spot from the console output (it was either going to be something kubernetes or… Dendron).
 
- Disable all extensions, reload
- Enable GitLive, reload (GitLive working)
- Enable Dendron, reload (GitLive not working).

So there you go. My first encounter with extension conflicts, I feel like now it will always happen :/
